deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
+
+Next, create the configmap that will be used for the Enriching Fluentd.
+
+First, generate a random key that will be used for communication between the two
+types of Fluentd pods:
+
+    $ tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 128
+
+Once we have the random secret key, save the following to your local directory as
+`enrichment-fluent.conf`:
+
+    @include configs.d/openshift/system.conf
+
+    ## source
+    <source>
+      @type secure_forward
+      @label @INGRESS
+
+      self_hostname ${HOSTNAME}-enrichment
+      bind 0.0.0.0
+      port 24284
+
+      shared_key <our generated string>
+
+      secure yes
+      ca_cert_path        /etc/fluentd/keys/ca
+      ca_private_key_path /etc/fluentd/keys/key
+    </source>
+
+    <label @INGRESS>
+    ## filters
+      @include configs.d/openshift/filter-pre-*.conf
+      @include configs.d/openshift/filter-retag-journal.conf
+      @include configs.d/openshift/filter-k8s-meta.conf
+      @include configs.d/openshift/filter-kibana-transform.conf
+      @include configs.d/openshift/filter-k8s-flatten-hash.conf
+      @include configs.d/openshift/filter-k8s-record-transform.conf
+      @include configs.d/openshift/filter-syslog-record-transform.conf
+      @include configs.d/openshift/filter-common-data-model.conf
+      @include configs.d/openshift/filter-post-*.conf
+    ##
+
+    ## matches
+      @include configs.d/openshift/output-operations.conf
+      @include configs.d/openshift/output-applications.conf
+    ##
+    </label>
+
+Then, create the configmap that will be used for the Enriching Fluentd, using the
+file created in the previous step as the source for the configmap:
+
+    $ oc create configmap logging-fluentd-enrichment \
+        --from-file=fluent.conf=enrichment-fluent.conf
+
+The next step will be to create the Deployment configuration that will be used to
+scale up the enrichment fluentd pods. Save the following file to your local file
+system as `enrichment-fluentd-dc.yaml`. Be sure to update the value of "K8S_HOST_URL"
+and "OPS_HOST" if necessary:
+
+    apiVersion: v1
+    kind: "DeploymentConfig"
+    metadata:
+      name: "logging-enrichment-fluentd"
+      labels:
+        provider: openshift
+        component: "fluentd"
+    spec:
+      replicas: 0
+      selector:
+        provider: openshift
+        component: "fluentd"
+      triggers: {}
+      strategy:
+        resources: {}
+        rollingParams:
+          intervalSeconds: 1
+          timeoutSeconds: 600
+          updatePeriodSeconds: 1
+        type: Recreate
+      template:
+        metadata:
+          name: enrichment-fluentd-elasticsearch
+          labels:
+            provider: openshift
+            component: "fluentd"
+        spec:
+          terminationGracePeriodSeconds: 300
+          serviceAccountName: aggregated-logging-fluentd
+          containers:
+          - name: enrichment-fluentd-elasticsearch
+            image: ${IMAGE_PREFIX}logging-fluentd:${IMAGE_VERSION}
```

Comment:
My thought with not providing a template specifically is that we may not make this a permanent solution, so it would be easier to clean up afterwards? I can be convinced otherwise though